### PR TITLE
added tests from rpmlint-tests (rpm-software-management#156) for FilesCheck

### PR DESCRIPTION
**Pull request by MeggyCal**

### CODE REVIEW

File: test/test_files.py [hunk 1]
```diff
@@ -185,3 +185,24 @@ def test_provides_devel(tmpdir, package, filescheck):
...
+
+
+@pytest.mark.parametrize('package', ['binary/shlib2-devel'])
+def test_shlib2_devel(tmpdir, package, filescheck):
+    output, test = filescheck
+    test.check(get_tested_package(package, tmpdir))
+    out = output.print_results(output.results)
+    assert 'library-without-ldconfig-postin' in out
+    assert 'library-without-ldconfig-postun' in out
+#    assert 'no-dependency-on' in out # this one currently fails
```
Comment (scarabeusiv) [inline test/test_files.py]: Please enable also the failing test we can check it up why it happens.

File: test/configs/testing-rpmlintrc [hunk 2]
```diff
@@ -33,7 +33,6 @@ addFilter("doc-file-dependency .*")
 addFilter("doc-file-dependency .*\.py ")addFilter("doc-file-dependency .*\.py ")addFilter("E: devel-file-in-non-devel-package .*site-packages.*fake_libc_include.*")
 addFilter("E: devel-file-in-non-devel-package")
 addFilter("E: devel-file-in-non-devel-package .*site-packages.*fake_libc_include.*")
-addFilter("E: shlib-policy-name-error")
```
Comment (scarabeusiv) [inline test/configs/testing-rpmlintrc]: This is just a list of random values not relevant to reality, why remove from here?

Comment (MeggyCal) [reply test/configs/testing-rpmlintrc]: OK, I have removed the second commit

File: test/test_files.py (hunk 1 above)
Comment (MeggyCal) [reply test/test_files.py]: ok

**Pull request merged**